### ISSUE TITLE
Accessibility: SizeOfSet and PositionInSet include offscreen RadioButtons items

### ISSUE THREAD

**Issue by gascardi** [microsoft/microsoft-ui-xaml#6785]
### Describe the bug

Offscreen children of a `RadioButtons` control are counted to calculate `SizeOfSet` and `PositionInSet`. This is an accessibility violation, since visually impaired users who rely on screen reader will face difficulty if narrator announces a position that does not correspond with the items they can interact with.

### Steps to reproduce the bug

1. Include a `<RadioButtons>` element with children in a page (for example, [XAML Control Gallery's RadioButtonPage.xaml](https://github.com/microsoft/Xaml-Controls-Gallery/blob/master/XamlControlsGallery/ControlPages/RadioButtonPage.xaml))

2. Change the visibility of one child to `Collapsed`. For example:
`<RadioButton Content="Option 1" Checked="RadioButton_Checked" Visibility="Collapsed" />`

3. Start the Narrator and toggle between the radio button items. Or launch Accessibility insights tool and inspect the radio button items on screen.

### Expected behavior

SizeOfSet and PositionInSet should only account for on-screen items.

For example, if there are 3 radio buttons, but the first one is offscreen:
- When focusing on the first radio button, Narrator should announce as 'Radio button, selected, 1 of 2'.
- When focusing on the second radio button, Narrator should announce as 'Radio button, selected, 2 of 2'.

### Screenshots

![image](https://user-images.githubusercontent.com/44008605/156640934-80d9a20c-312d-4611-81af-f28c0c8b2cf3.png)

### NuGet package version

Microsoft.UI.Xaml 2.7.0

### Windows app type

- [ ] UWP
- [ ] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (21H2): Build 22000, Windows 10 (21H2): Build 19044

### Additional context

We have a use case for hiding radio buttons when a feature is flighted off.

In the past we had the same problem with list view items, but we found that assigning a style with those fields as "0" forced the automation peer to re-evaluate and return the correct values:
```
<Style TargetType="ListViewItem">
    <Setter Property="AutomationProperties.PositionInSet" Value="0" />
    <Setter Property="AutomationProperties.SizeOfSet" Value="0" />
</Style>
```

Unfortunately, the same does not apply to the `RadioButton` children of the `RadioButtons` control.

Thank you for your assistance.

**Comment by StephenLPeters** [comment]
>This is an accessibility violation, since visually impaired users who rely on screen reader will face difficulty if narrator announces a position that does not correspond with the items they can interact with.

This isn't quite accurate is it?  If the radio button was disabled you would still want it included in the count correct?  I guess there is some amount of narrator interaction that is allowed with disabled elements.
